ios/FluentUI/Card Nudge/CardNudge.swift

```diff
@@ -46,7 +46,7 @@ public typealias CardNudgeButtonAction = ((_ state: MSFCardNudgeState) -> Void)
 }
 
 /// View that represents the CardNudge.
-public struct CardNudge: TokenizedControlInternal {
+public struct CardNudge: View, TokenizedControlInternal {
```

Comment:
for my own knowledge, why is the View needed?

Reply:
Because I removed it from `TokenizedControlInternal`, so I could apply that protocol to `UIView` as well.